invokeai/backend/stable_diffusion/diffusers_pipeline.py

```diff
@@ -533,8 +534,15 @@ def step(
         noise_pred = self.invokeai_diffuser._combine(
             uc_noise_pred,
             c_noise_pred,
-            guidance_scale,
+            guidance_scale
         )
+        guidance_rescale_multiplier = conditioning_data.guidance_rescale_multiplier
+        if guidance_rescale_multiplier > 0:
+            noise_pred = type(self).rescale_cfg(
```

Comment:
There no need to extract type, classmethods will work correctly being called on self too.
Also you can make it here staticmethod.
// staticmethods too can be called normally on self.

Reply:
done